darwin/importer/formats/nifti.py

```diff
@@ -531,9 +549,16 @@ def process_nifti(
     img = correct_nifti_header_if_necessary(input_data)
     orig_ax_codes = nib.orientations.aff2axcodes(img.affine)
     orig_ornt = nib.orientations.axcodes2ornt(orig_ax_codes)
+    if remote_file_path in remote_files_that_require_legacy_scaling:
+        is_dicom = remote_file_path.suffix.lower() == ".dcm"
+        slot_affine_map = remote_files_that_require_legacy_scaling[remote_file_path]
+        affine = slot_affine_map[next(iter(slot_affine_map))]  # Take the 1st slot
+        ax_codes = nib.orientations.aff2axcodes(affine)
+        ornt = nib.orientations.axcodes2ornt(ax_codes)
+        if not is_dicom:
+            img = nib.Nifti1Image(np.flip(img.get_fdata(), (0, 1, 2)), affine)
```

Comment:
I think it might be more robust to do `as_closest_canonical` on `img` to get it to RAS, since this will only work if the Nifti annotation file is in LPI right

Reply:
Updated